Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Footer.css';
+
+function Icon (props) {
+    return (
+        <div className='icon-container'>
+            <img src={props.src} className='icon-image' alt='' />
+            <p className='footer-text'>{props.text}</p>
+        </div>
+    )
+}
+
+function Footer (props) {
+    return (
+        <footer>
+            <Icon src={footerIconsSrc[0]} text={footerText[0]} />
```

Comment:
Same comment as before, take some time to think about how you could better display your data in a more dynamic fashion.